### DISCUSSION TITLE
How is the proper way to add config to a cluster

### DISCUSSION THREAD

**Discussion by frit0-rb** [rancher/rke2#5363] [Q&A]
Good afternoon everybody,

I want to increase the event time for a Cluster.

Like I see in this doc https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/#:~:text=%2D%2Devent%2Dttl%20duration%C2%A0%C2%A0%C2%A0%C2%A0%C2%A0Default%3A%201h0m0s is it possible with that command.

I got two Cluster, one in Rancher and another Imported

Rancher Cluster: My config is located in vim /etc/rancher/rke2/config.yaml.d/50-rancher.yaml  and I got some config there, but I dont know how to add config to that file ( im so new in kubernetes)

This is the file.

`{
  "node-label": [
    "cattle.io/os=linux",
    "rke.cattle.io/machine=5dbe810a-f09a-4939-94d0-8639b95b43ac"
  ],
  "private-registry": "/etc/rancher/rke2/registries.yaml",
  "protect-kernel-defaults": false,
  "server": "https://k8sc.example.lan:9345",
  "token": "4kb7v7gq6hf44wq445klwmc2krwxq66jcgkchlvdcgmtfwwqn2gtkg",
  "kubelet-arg": [
    "image-gc-high-threshold=75",
    "image-gc-low-threshold=60"
  ],
  "kube-apiserver-arg": [
    "event-ttl=24h"
  ]
}
`

I got another cluster in RKE2  and the server cluster (controller) with the config in /etc/rancher/rke2/config.yaml and only got this config:

`selinux: true`

I want to know hot to add the flag event-ttl to both cluster. I got more qestions?

How I know this flags are only for servers and not for agents?

How I know write the config in that file?

In the doc describe the flag but there aren´t any example, and dont say if it is only for server o for agents

Thanks a lot

**Comment by brandond** [comment]
> My config is located in vim /etc/rancher/rke2/config.yaml.d/50-rancher.yaml and I got some config there

Don't edit that. This configuration file is managed by Rancher. If your cluster was provisioned by Rancher, you should manage the configuration through Rancher.

For the provisioned cluster, see the docs at https://docs.rke2.io/install/configuration
